exonum/src/node/consensus.rs

```diff
@@ -576,6 +559,35 @@ impl NodeHandler {
             self.remove_request(&RequestData::Transactions(hash));
             self.has_full_propose(hash, round);
         }
+        Ok(())
+    }
+
+    /// Handles raw transaction. Transaction is ignored if it is already known, otherwise it is
+    /// added to the transactions pool.
+    #[cfg_attr(feature = "flame_profile", flame)]
+    pub fn handle_tx(&mut self, msg: RawTransaction) {
+        //trace!("Handle transaction");
```

Comment:
I think such commented code should be removed.

Reply:
I didn't remove this on purpose. From my point of view it's kind of strange that handle_incoming_tx contains trace, and handle_tx has it commented.  Since they are similar, wouldn't it be more logical to either both have or both not have traces?